sunrise.el

```diff
@@ -1415,7 +1419,22 @@ If FILENAME is non-nil, it is the basename of a file to focus."
 (defun sunrise-toggle ()
   "Show or hide the Sunrise Commander."
   (interactive)
-  (or (sunrise-quit) (sunrise-show)))
+  (or (sunrise-quit) (sunrise-setup t)))
+
+(defun sunrise-setup-windows ()
```

Comment:
This function used to exist before the refactor and parts of the code still call it... I tried to re-add it back in as a way to repaint the panes. I suspect there is a better way to repaint panes instead of essentially restarting Sunrise, but that's the hack I did for now.